homeassistant/components/yeelight/__init__.py

```diff
@@ -316,12 +316,29 @@ async def async_unload_entry(hass: HomeAssistant, entry: ConfigEntry) -> bool:
     return True
 
 
+@callback
+def async_format_model(model: str) -> str:
+    """Generate a more human readable model."""
+    return str(model).replace("_", " ").title()
+
+
+@callback
+def async_format_id(id_: str) -> str:
+    """Generate a more human readable id."""
+    return str(hex(int(id_, 16))) if id_ else "None"
```

Comment:
```suggestion
    return hex(int(id_, 16)) if id_ else "None"
```
`hex` returns a string already, so no need to cast it separately.